go/arrow/flight/flightsql/types.go

```diff
@@ -854,3 +880,19 @@ const (
 )
 
 type CreatePreparedStatementResult = pb.ActionCreatePreparedStatementResult
+
+type TableDefinitionOptions = pb.CommandStatementIngest_TableDefinitionOptions
+
+type TableDefinitionOptionsTableNotExistOption = pb.CommandStatementIngest_TableDefinitionOptions_TableNotExistOption
+type TableDefinitionOptionsTableExistsOption = pb.CommandStatementIngest_TableDefinitionOptions_TableExistsOption
```

Comment:
for clarity should we gather these into a single declaration block?

```suggestion
type (
        CreatePreparedStatementResult                     = pb.ActionCreatePreparedStatementResult
        TableDefinitionOptions                                      = pb.CommandStatementIngest_TableDefinitionOptions
        TableDefinitionOptionsTableNotExistOption = pb.CommandStatementIngest_TableDefinitionOptions_TableNotExistOption
        TableDefinitionOptionsTableExistsOption      = pb.CommandStatementIngest_TableDefinitionOptions_TableExistsOption
)
```

?

Reply:
Sure, I gathered the ones related to TableDefinitionOptions.

The CreatePreparedStatementResult type is unrelated.